Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -2917,10 +2954,8 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>, publ
     bool m_isListeningForUserFacingStateChangeNotification { false };
 #endif
     bool m_allowsMediaDocumentInlinePlayback { false };
-    std::unique_ptr<ProcessThrottlerActivity> m_isVisibleActivity;
-    std::unique_ptr<ProcessThrottlerActivity> m_isAudibleActivity;
-    std::unique_ptr<ProcessThrottlerActivity> m_isCapturingActivity;
-    std::unique_ptr<ProcessThrottlerActivity> m_openingAppLinkActivity;
+
+    ProcessActivityState m_processActivityState;
```

Comment:
My earlier comment about not needed the unique_ptr only applied to this data member.